src/sqlite3/connection.cr

```diff
@@ -87,3 +89,11 @@ class SQLite3::Connection < DB::Connection
     raise Exception.new(self) unless code == 0
   end
 end
+
+REGEXP_FN = ->(context : LibSQLite3::SQLite3Context, argc : Int32, argv : LibSQLite3::SQLite3Value*) do
```

Comment:
Great, but let's leave this constant within SQLite3 namespace.

```cr
module SQLite3
  # :nodoc:
  REGEXP_FN = ->(context : ...
end
```

It's better if placed either at the beginning of this file or at `src/sqlite3.cr`. Either will work.

Other than that, LGTM!

Reply:
Thanks! I put it in `src/sqlite3.cr`.